db-service/test/cqn4sql/flattening.test.js

```diff
@@ -409,9 +440,15 @@ describe('Flattening', () => {
     })
 
     it('rejects managed associations in expressions in WHERE clause (2)', () => {
-      expect(() => cqn4sql(cds.ql`SELECT from bookshop.Books { ID } WHERE sin(author) < 0`, model)).to.throw(
-        /An association can't be used as a value in an expression/,
-      )
+      const transformed = cqn4sql(cds.ql`SELECT from bookshop.Books { ID } WHERE sin(author) < 0`, model)
```

Comment:
again: like above - ok for single elem, but not in general

Reply:
rejected